cmd/generate/config/rules/sentry.go

```diff
@@ -1,16 +1,18 @@
 package rules
 
 import (
+	"encoding/base64"
 	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
 	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
 	"github.com/zricethezav/gitleaks/v8/config"
+	"regexp"
 )
 
 func SentryAccessToken() *config.Rule {
 	// define rule
 	r := config.Rule{
 		RuleID:      "sentry-access-token",
-		Description: "Found a Sentry Access Token, risking unauthorized access to error tracking services and sensitive application data.",
+		Description: "Found a Sentry.io Access Token (old format), risking unauthorized access to error tracking services and sensitive application data.",
 		Regex:       utils.GenerateSemiGenericRegex([]string{"sentry"}, utils.Hex("64"), true),
 
```

Comment:
While we're here, can we add entropy to this rule as well?
```suggestion
Entropy: 3,
```